nca/nca_cli.py

```diff
@@ -326,6 +327,9 @@ def nca_main(argv=None):
     parser.add_argument('--file_out', '-f', type=str, help='A file path to which output is redirected')
     parser.add_argument('--expected_output', type=str, help='A file path of the expected query output,'
                                                             'relevant only with --connectivity and --semantic_diff')
+    parser.add_argument('--simplify_output', action='store_true',
+                        help='simplify the connectivity graph,'
+                             'relevant only when output_format is dot or jpg')
```

Comment:
- consider renaming the flag `simplify_output` to be more specific, such as `simplify_graph`
- add documentation about the new flag option on `md` files (main readme that lists all flgas, and scheme file options that details outputConfiguration options).

Reply:
done

Reply:
done